src/test/regress/sql/foreign_data.sql

```diff
@@ -544,7 +544,54 @@ REASSIGN OWNED BY regress_test_role TO regress_test_role2;
 DROP OWNED BY regress_test_role2;
 DROP OWNED BY regress_test_role2 CASCADE;
 
+------------------------------------------------------------------------------
+-- Test to check column encoding at foreign tables.
+-- Not use column encoding at foreign tables.
+------------------------------------------------------------------------------
+
+create database foreign_tables_test_db;
+\c foreign_tables_test_db
+
+create extension postgres_fdw;
```

Comment:
Take a look around other adb regress tests. We don't use postgres_fdw not just like that. It's a separate software from contrib with own set of tests.
In my last comment I said we have a dummy data wrapper here.